back/src/chat/chat.gateway.ts

```diff
@@ -28,5 +36,11 @@ export class ChatGateway implements OnGatewayInit, OnGatewayConnection {
 		const user = await this.authService.getWsUser(socket);
 		if (!user) socket.disconnect(true);
 		else await this.chatService.onConnection(socket);
+		this.chatService.emit(`user:${user.id}:status`, await this.statusService.getByUserId(user.id));
+	}
+
+	async handleDisconnect(socket: Socket): Promise<void> {
+		if (!socket.user) return;
+		this.chatService.emit(`user:${socket.user.id}:status`, await this.statusService.getByUserId(socket.user.id));
```

Comment:
So you also made that to prepare the field for online status, right?

Reply:
Yeah, in fact i implemented a basic status management in this PR.
For now, the offline and online status are already implemented and working.